au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml

```diff
@@ -18,170 +18,49 @@ StyledPopupView {
-                    if (prv.numberOfTracks === "") {
-                        input.focus = false
-                        return
+            spacing: 12
+
+            Repeater {
+                model: [
+                    { type: TrackType.MONO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Mono") },
+                    { type: TrackType.STEREO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Stereo") },
+                    { type: TrackType.LABEL, icon: IconCode.LOOP_IN, text: qsTrc("projectscene", "Label") }
+                ]
+
+                FlatButton {
+                    Layout.preferredWidth: 64
+                    Layout.preferredHeight: 64
+                    Layout.fillHeight: true
+                    accentButton: false
+                    enabled: true
+                    icon: modelData.icon
+                    text: modelData.text
+                    onClicked: {
+                        createTrack(modelData.type)
                     }
-
-                    var value = Number(prv.numberOfTracks)
-                    if (value > numberOfTracksValidator.top) {
-                        prv.numberOfTracks = numberOfTracksValidator.top
-                    } else if (value > numberOfTracksValidator.bottom) {
-                        prv.numberOfTracks = value - 1
-                    }
-
-                    input.focus = false
-                }
-            }
-
-            TextInputField {
-                id: input
-
-                Layout.preferredWidth: 79
-                Layout.preferredHeight: 28
-
-                clip: true
-
-                currentText: prv.numberOfTracks
-                textHorizontalAlignment: Qt.AlignHCenter
-                textVerticalAlignment: Qt.AlignVCenter
-
-                validator: IntValidator {
-                    id: numberOfTracksValidator
-                    bottom: 1
-                    top: 25
-                }
-
-                onTextEditingFinished: function(newTextValue) {
-                    var value = Number(newTextValue)
-                    prv.numberOfTracks = value
-                }
-            }
-
-            FlatButton {
-                id: add
-
-                Layout.preferredWidth: 28
-                Layout.preferredHeight: 28
-
-                text: "+"
-
-                onClicked: {
-                    if (prv.numberOfTracks === "") {
-                        input.focus = false
-                        return
-                    }
-
-                    var value = Number(prv.numberOfTracks)
-                    if (value < numberOfTracksValidator.bottom) {
-                        prv.numberOfTracks = numberOfTracksValidator.bottom
-                    } else if (value < numberOfTracksValidator.top) {
-                        prv.numberOfTracks = value + 1
-                    }
-
-                    input.focus = false
                 }
             }
         }
 
         SeparatorLine {}
```

Comment:
don't know how important is that but on the design SeparatorLine stretches exactly from left flyout edge to the right (right now it has a few pixels margin)

Reply:
The design is the design. I'll address.